src/app/view/components/dot-persona-selector-option/dot-persona-selector-option.component.ts

```diff
@@ -41,7 +41,7 @@ export class DotPersonaSelectorOptionComponent implements OnInit {
                             messages['modes.persona.personalized']
                     )
                 );
-        }, 0);
+        }, 10);
```

Comment:
why?

Reply:
Weird things with the change detector when you're getting the message key in a component that your're passing as template.